src/index.html

```diff
@@ -1,57 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      href="https://pro.fontawesome.com/releases/v5.15.1/css/all.css"
-    />
-    <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>To-Do-List</title>
+    <meta charset="UTF-8">  
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="style.css" rel="stylesheet">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">    
```

Comment:
- [ ] Please make sure to fix this linter error
![image](https://user-images.githubusercontent.com/74744787/185767638-d304236b-db13-43a5-959f-d1f3fc80f605.png)